app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
Lets convert all this to HAML instead of ERB files. 

Sorry, I didn't add this in the original scope.

Gem to install and convert: https://github.com/haml/haml-rails

Learning HAML: https://haml.info